ci/lint.sh

```diff
@@ -125,6 +125,16 @@ if [ "$LINT" ]; then
         RET=1
     fi
     echo "Check for deprecated messages without sphinx directive DONE"
+
```

Comment:
put this in a separete script (ci/asv.sh)

and we can enable it in (.travis.yml) via an ENV variable (e.g. like we do for coverage).

Then its easy to put it on only 1 build.

also need to install asv (on that build)